cmd/kubeadm/app/master/manifests.go

```diff
@@ -221,88 +221,100 @@ func componentPod(container api.Container, volumes ...api.Volume) api.Pod {
 	}
 }
 
-func getComponentCommand(component string, cfg *kubeadmapi.MasterConfiguration) (command []string) {
-	baseFlags := map[string][]string{
-		apiServer: {
-			"--insecure-bind-address=127.0.0.1",
-			"--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota",
-			"--service-cluster-ip-range=" + cfg.Networking.ServiceSubnet,
-			"--service-account-key-file=" + pkiDir + "/apiserver-key.pem",
-			"--client-ca-file=" + pkiDir + "/ca.pem",
-			"--tls-cert-file=" + pkiDir + "/apiserver.pem",
-			"--tls-private-key-file=" + pkiDir + "/apiserver-key.pem",
-			"--token-auth-file=" + pkiDir + "/tokens.csv",
-			fmt.Sprintf("--secure-port=%d", cfg.API.BindPort),
-			"--allow-privileged",
-		},
-		controllerManager: {
-			"--address=127.0.0.1",
-			"--leader-elect",
-			"--master=127.0.0.1:8080",
-			"--cluster-name=" + DefaultClusterName,
-			"--root-ca-file=" + pkiDir + "/ca.pem",
-			"--service-account-private-key-file=" + pkiDir + "/apiserver-key.pem",
-			"--cluster-signing-cert-file=" + pkiDir + "/ca.pem",
-			"--cluster-signing-key-file=" + pkiDir + "/ca-key.pem",
-			"--insecure-experimental-approve-all-kubelet-csrs-for-group=system:kubelet-bootstrap",
-		},
-		scheduler: {
-			"--address=127.0.0.1",
-			"--leader-elect",
-			"--master=127.0.0.1:8080",
-		},
-		proxy: {},
-	}
-
+func getComponentBaseCommand(component string) (command []string) {
 	envParams := kubeadmapi.GetEnvParams()
 	if envParams["hyperkube_image"] != "" {
 		command = []string{"/hyperkube", component}
 	} else {
-		command = []string{"/usr/local/bin/kube-" + component}
+		command = []string{"kube-" + component}
```

Comment:
Why was this leading path removed? Should the path to hyperkube above still be absolute?